articles/openshift/oshift-faq.md

```diff
@@ -77,7 +79,9 @@ We provide confidence that our OpenShift service still meets the highest level o
 
 ### Can I use UKCloud for OpenShift over closed networks such as PSN and HSCN?
 
-UKCloud for OpenShift is available on the PSN, HSCN and Janet networks.
+UKCloud for OpenShift has been certified for use over both the PSN Assured and Protected networks.
```

Comment:
I don't think we should be referring to PSN Assured and PSN Protected anymore now that the network has merged it's all just PSN. So may "certified for use over the PSN network in both our Assured and Elevated security domains."

Reply:
Updated accordingly